src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -4,63 +4,107 @@ import {
   forwardRef,
   HTMLAttributes,
   useLayoutEffect,
+  useRef,
 } from 'react';
 import ReactDOM from 'react-dom';
-import { useThree } from '@react-three/fiber';
+import { GroupProps, useFrame, useThree } from '@react-three/fiber';
+import { Group, Vector3 } from 'three';
 
 // Simplified version of `drei`'s `<Html>` component
-// https://github.com/pmndrs/drei/blob/v2.2.3/src/Html.tsx
-const Html = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
-  (props, ref) => {
-    const { className, style, children, ...divProps } = props;
+// https://github.com/pmndrs/drei/blob/v6.0.3/src/web/Html.tsx
 
-    const { width, height } = useThree((state) => state.size);
-    const gl = useThree((state) => state.gl);
-    const { parentElement } = gl.domElement;
+const v1 = new Vector3();
 
-    // Container `div` for ReactDOM to render into, appended next to R3F's `canvas`
-    const [el] = useState(() => document.createElement('div'));
+export interface HtmlProps extends HTMLAttributes<HTMLDivElement> {
+  groupProps?: GroupProps;
+  followCamera?: boolean;
+  fullscreen?: boolean;
+}
 
-    useEffect(() => {
-      el.style.cssText = `position: absolute; top: 0; left: 0;`;
+const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
+  const {
+    className,
+    style,
+    children,
+    followCamera,
+    fullscreen,
+    groupProps = {},
+    ...divProps
+  } = props;
 
+  const size = useThree((state) => state.size);
+  const gl = useThree((state) => state.gl);
+  const { parentElement } = gl.domElement;
+  const camera = useThree(({ camera }) => camera);
+
+  // Container `div` for ReactDOM to render into, appended next to R3F's `canvas`
+  const [el] = useState(() => document.createElement('div'));
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Maybe add a small comment here to explain what the group is for (i.e. for `followCamera` / position tracking).